STROOP/Managers/M64Manager.cs

```diff
@@ -40,6 +40,10 @@ public M64Manager(M64Gui gui)
             _gui.ButtonCopyRom.Click += (sender, e) => CopyHeaderRomVersion();
             _gui.ButtonPasteRom.Click += (sender, e) => PasteHeaderRomVersion();
 
+            _m64File = new M64File(_gui);
```

Comment:
Why the move? (Just curious?)

Reply:
Uhh it doesn't seem to be for anything? I thought the `SelectionChanged` handler was being triggered and failing due to something being uninitialised, and this change (https://github.com/SM64-TAS-ABC/STROOP/pull/113/commits/9937d36291f2fc6955a2b49028e855a3eb1a6f0c) was to fix that, but I'm no longer seeing that stacktrace. That might be worth looking into anyway.

Reply:
Guess it doesn't harm anything keeping